src/specs/api.v2.yaml

```diff
@@ -1779,13 +1779,12 @@ paths:
             type: string
           in: query
           required: true
-  "/workResults/{experimentId}/{ETag}":
-    get:
-      summary: Get the work results from S3
-      operationId: getWorkResults
-      x-eov-operation-id: work#getResults
```

Comment:
Endpoint is no longer necessary

Reply:
Some context [here](https://github.com/hms-dbmi-cellenics/ui/pull/885#discussion_r1313381913)